Can't build test library locally

When checking out the current head on master I can't build the library as there are test failures.

```
ERROR] Failures:
[ERROR]   MessageAssertTest$HappyPathTests.testHasExpired:108 Message with key 2251799813685251 has not expired
[ERROR]   MessageAssertTest$UnhappyPathTests.testHasNotExpiredFailure:274
Expecting code to raise a throwable.
[ERROR]   ProcessInstanceAssertTest$UnhappyPathTests.testProcessInstanceHasPassedElementsInOrderFailure:698
Expecting message to be:
  "[Ordered elements]
expected: ["endevent", "servicetask", "startevent"]
 but was: ["startevent", "servicetask", "endevent"]"
but was:
  "[Ordered elements]
expected: ["endevent", "servicetask", "startevent"]
 but was: ["startevent", "servicetask", "endevent"]"

Throwable that failed the check:

org.opentest4j.AssertionFailedError: [Ordered elements]
expected: ["endevent", "servicetask", "startevent"]
 but was: ["startevent", "servicetask", "endevent"]
        at jdk.internal.reflect.GeneratedConstructorAccessor41.newInstance(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
        at io.camunda.zeebe.process.test.assertions.ProcessInstanceAssert.hasPassedElementInOrder(ProcessInstanceAssert.java:223)
        at io.camunda.zeebe.process.test.assertions.ProcessInstanceAssertTest$UnhappyPathTests.lambda$testProcessInstanceHasPassedElementsInOrderFailure$8(ProcessInstanceAssertTest.java:693)
        at io.camunda.zeebe.process.test.assertions.ProcessInstanceAssertTest$UnhappyPathTests.testProcessInstanceHasPassedElementsInOrderFailure(ProcessInstanceAssertTest.java:690)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:564)
        at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
        at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
        at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
        at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
        at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
        at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
        at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)

[ERROR]   ProcessEventInspectionsTest.testFindFirstProcessInstance:40
Expecting Optional to contain a value but it was empty.
[ERROR]   ProcessEventInspectionsTest.testFindLastProcessInstance:60
Expecting Optional to contain a value but it was empty.
[INFO]
[ERROR] Tests run: 185, Failures: 5, Errors: 0, Skipped: 0
```

Same issue on that branch:

```
[INFO] Tests run: 19, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.042 s - in io.camunda.zeebe.process.test.testengine.EngineClientTest
11:57:45.524 [-zb-fs-workers-1] DEBUG io.camunda.zeebe.util.actor - Closing actor thread ground '-zb-fs-workers': closed successfully
11:57:45.524 [-zb-actors-4] DEBUG io.camunda.zeebe.util.actor - Closing actor thread ground '-zb-actors': closed successfully
[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   MessageAssertTest$UnhappyPathTests.testHasNotExpiredFailure:274
Expecting code to raise a throwable.
[ERROR]   ProcessInstanceAssertTest$UnhappyPathTests.testProcessInstanceHasPassedElementsInOrderFailure:698
Expecting message to be:
  "[Ordered elements]
expected: ["endevent", "servicetask", "startevent"]
 but was: ["startevent", "servicetask", "endevent"]"
but was:
  "[Ordered elements]
expected: ["endevent", "servicetask", "startevent"]
 but was: ["startevent", "servicetask", "endevent"]"

Throwable that failed the check:

org.opentest4j.AssertionFailedError: [Ordered elements]
expected: ["endevent", "servicetask", "startevent"]
 but was: ["startevent", "servicetask", "endevent"]
```

--- a/src/main/java/io/camunda/zeebe/process/test/assertions/IncidentAssert.java
+++ b/src/main/java/io/camunda/zeebe/process/test/assertions/IncidentAssert.java
@@ -18,6 +18,7 @@ import org.assertj.core.api.StringAssert;
 
 /** Assertions for incidents. An incident is identified by its incident key. */
 public class IncidentAssert extends AbstractAssert<IncidentAssert, Long> {
+  private final String LINE_SEPARATOR = System.lineSeparator();
 
   private final RecordStreamSource recordStreamSource;
 
@@ -226,34 +227,34 @@ public class IncidentAssert extends AbstractAssert<IncidentAssert, Long> {
     final Optional<Record<IncidentRecordValue>> optRecord = findIncidentCreatedRecord();
 
     if (!optRecord.isPresent()) {
-      return "\nNo incident details found for key " + actual;
+      return LINE_SEPARATOR + "No incident details found for key " + actual;
     } else {
       final Record<IncidentRecordValue> record = optRecord.get();
 
       final StringBuilder result = new StringBuilder();
       result
-          .append("\nIncident[")
-          .append("\n  key: ")
+          .append(LINE_SEPARATOR + "Incident[")
+          .append(LINE_SEPARATOR + "  key: ")
           .append(record.getKey())
-          .append("\n  errorType: ")
+          .append(LINE_SEPARATOR + "  errorType: ")
           .append((record.getValue().getErrorType()))
-          .append("\n  errorMessage: \"")
+          .append(LINE_SEPARATOR + "  errorMessage: \"")
           .append(record.getValue().getErrorMessage())
           .append("\"");
 
       final String elementId = record.getValue().getElementId();
 
       if (elementId != null) {
-        result.append("\n  elementId: ").append(elementId);
+        result.append(LINE_SEPARATOR + "  elementId: ").append(elementId);
       }
 
       final long jobKey = record.getValue().getJobKey();
 
       if (jobKey != -1) {
-        result.append("\n  jobKey: ").append(jobKey);
+        result.append(LINE_SEPARATOR + "  jobKey: ").append(jobKey);
       }
 
-      result.append("\n]");
+      result.append(LINE_SEPARATOR + "]");
       return result.toString();
     }
   }

--- a/src/test/java/io/camunda/zeebe/process/test/assertions/ProcessInstanceAssertTest.java
+++ b/src/test/java/io/camunda/zeebe/process/test/assertions/ProcessInstanceAssertTest.java
@@ -24,6 +24,7 @@ import org.junit.jupiter.api.Test;
 
 @ZeebeProcessTest
 class ProcessInstanceAssertTest {
+  private static final String LINE_SEPARATOR = System.lineSeparator();
   private static final Map<String, Object> TYPED_TEST_VARIABLES = new HashMap<>();
 
   static {
@@ -696,8 +697,10 @@ class ProcessInstanceAssertTest {
                           ProcessPackLoopingServiceTask.START_EVENT_ID))
           .isInstanceOf(AssertionError.class)
           .hasMessage(
-              "[Ordered elements] \n"
-                  + "expected: [\"endevent\", \"servicetask\", \"startevent\"]\n"
+              "[Ordered elements] "
+                  + LINE_SEPARATOR
+                  + "expected: [\"endevent\", \"servicetask\", \"startevent\"]"
+                  + LINE_SEPARATOR
                   + " but was: [\"startevent\", \"servicetask\", \"endevent\"]");
     }
 

